website/docs/getting-started/setup.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 0
+---
+
+# Setup
+
+Functionless requires a few prerequisite setup steps to ensure your environment
+is ready to build and deploy the code you write as infrastructure to AWS.
```

Comment:
nit: pre-requisite

Also, curious why there's a new line in the middle of the sentence? Markdown will automatically wrap. 

I've seen some projects where each sentence has a new line since two sentences on subsequent lines will actually be collapsed to a single line when rendered. They do this so that each sentence can be commented on individually in a pull request. 

Curious what your thoughts are?